engine/src/main/java/org/terasology/rendering/shader/ShaderParametersCombine.java

```diff
@@ -107,6 +107,12 @@ public void applyParameters(Material program) {
             program.setFloat("outlineThickness", outlineThickness, true);
         }
 
+        //TODO: Other parameters and volumetric fog test case is needed
+        Camera activeCamera = CoreRegistry.get(WorldRenderer.class).getActiveCamera();
+        if (activeCamera != null && CoreRegistry.get(Config.class).getRendering().isVolumetricFog()) {
+            program.setMatrix4("invViewProjMatrix", activeCamera.getInverseViewProjectionMatrix(), true);
```

Comment:
In https://github.com/tdgunes/Terasology/blob/new-options/engine/src/main/resources/assets/shaders/combine_frag.glsl line 54, I suppose if volumetric fog is enabled, the implementation requires invViewProjMatrix to be supplied by "activeCamera's inverse view projection matrix". But since, I couldn't figure out a way to test volumetric fog, for now this is useless. :(